lib/src/optimizely_client_wrapper.dart

```diff
@@ -53,6 +54,7 @@ class OptimizelyClientWrapper {
   static Map<String, Map<int, LogEventNotificationCallback>>
       logEventCallbacksById = {};
   static Map<String, Map<int, MultiUseCallback>> configUpdateCallbacksById = {};
+  static String sdkVersion =  "";
```

Comment:
what is this for in addition to one in line 71?

Reply:
That's is my bad, will remove it.